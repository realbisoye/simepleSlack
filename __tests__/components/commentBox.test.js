import 'react-native'
import React from 'react'
import CommentBox from '../../app/components/commentBox'

import renderer from 'react-test-renderer'

const addMessage = jest.fn()
it('renders CommentBox component correctly', () => {
  const rendered = renderer.create(
    <CommentBox addMessage={addMessage} />
  )
  expect(rendered).toMatchSnapshot()
})
