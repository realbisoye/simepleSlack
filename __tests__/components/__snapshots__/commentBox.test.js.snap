// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CommentBox component correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "borderColor": "#DCDDDF",
      "borderTopWidth": 0.5,
      "bottom": 0,
      "flex": 1,
      "flexDirection": "column",
      "maxHeight": 100,
      "padding": 10,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "space-around",
        "marginBottom": 5,
        "width": "100%",
      }
    }
  >
    <Icon
      color="#808080"
      name="insert-emoticon"
      size={25}
    />
    <Icon
      color="#808080"
      name="photo-camera"
      size={25}
    />
    <Icon
      color="#808080"
      name="insert-photo"
      size={25}
    />
    <Icon
      color="#808080"
      name="attach-file"
      size={25}
    />
    <Icon
      color="#808080"
      name="crop-3-2"
      size={25}
    />
    <Icon
      color="#808080"
      name="email"
      size={25}
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "space-between",
        "paddingBottom": 10,
        "width": "100%",
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      autoCapitalize="sentences"
      multiline={true}
      onChangeText={[Function]}
      placeholder="Reply"
      placeholderTextColor="#A9A9A9"
      rejectResponderTermination={true}
      style={
        Object {
          "flex": 1,
          "maxHeight": 120,
        }
      }
      underlineColorAndroid="transparent"
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <Icon
        color="#DCDDDF"
        name="send"
        size={30}
      />
    </View>
  </View>
</View>
`;
